src/scalarstats.jl

```diff
@@ -257,6 +334,14 @@ zscore!{T<:AbstractFloat}(X::AbstractArray{T}, μ::Real, σ::Real) = _zscore!(X,
 zscore!{T<:AbstractFloat,U<:Real,S<:Real}(X::AbstractArray{T}, μ::AbstractArray{U}, σ::AbstractArray{S}) =
     (_zscore_chksize(X, μ, σ); _zscore!(X, X, μ, σ))
 
+
+"""
+    zscore(X, [μ,] [σ]) -> Array
```

Comment:
The comma should either be outside of brackets or inside the last pair of brackets. You should be able to do a mass replacement.